src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name 'AreaCalculator' is a noun, which is good. However, make sure that this interface is used only for calculating areas, as per the checklist item: 'Don't put all behavior into a single interface if the methods are conceptually different from each other.'